### PR TITLE
Better cloning flavour text

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -335,10 +335,8 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		clonemind.transfer_to(occupant)
 		occupant.grab_ghost()
-		occupant << "<span class='notice'><b>The world is suddenly bright \
-			and sudden and loud!</b><br>\
-			<i>You feel your body weight suddenly, as your mind suddenly \
-			comprehends where you are and what is going on.</i></span>"
+		occupant << "<span class='notice'><b>There is a bright flash</b><br>\
+			<i>You feel like a new human</i></span>"
```
Comment (Cheridan) [inline code/game/machinery/cloning.dm]: punctuationnnn

**Comment by WJohn** [comment]
WHAT IF YOU'RE A LIZARDMAN THOUGH

WOULD YOU STILL FEEL LIKE A NEW HUMAN THEN?

**Comment by optimumtact** [comment]
Yes

**Comment by Cheridan** [comment]
Suddenly, I'm suddenly going to suddenly merge this all of a sudden.

**Comment by Cheridan** [comment]
That's not the button.

**Comment by Alek2ander** [comment]
As a nulldimensional transcendental nonentity I am triggered

**Pull request merged**